drivers/Makefile.include

```diff
@@ -28,6 +28,10 @@ ifneq (,$(filter ata8520e,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/ata8520e/include
 endif
 
+ifneq (,$(filter at24mac,$(USEMODULE)))
```

Comment:
I think this block should be moved before the at86rf2xx block (sorry, alphabetical order...).

Reply:
Order restored.